terasoluna-gfw-common-libraries/terasoluna-gfw-validator/src/test/java/org/terasoluna/gfw/common/validator/constraints/AbstractConstraintsTest.java

```diff
@@ -67,6 +67,20 @@ protected void setExpectedFailedToInitialize(Class<?> cls) {
     }
 
     /**
+     * set {@code ExpectedException} for failed to initialize.
+     * @param cls expected inner exception.

```

Comment:
- [ ] Probably argument variable name should be better like `causeType` rather than `cls`.